src/Microsoft.DotNet.Build.Tasks/Targets/localization.targets

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Label="MultilingualAppToolkit">
+    <MultilingualAppToolkitVersion>4.0</MultilingualAppToolkitVersion>
+    <MultilingualFallbackLanguage>en-US</MultilingualFallbackLanguage>
+  </PropertyGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="$(MSBuildProjectDirectory)\Resources\Strings.de.resx">
+      <Visible>True</Visible>
+    </EmbeddedResource>
+    <EmbeddedResource Include="$(MSBuildProjectDirectory)\Resources\Strings.es.resx">
+      <Visible>True</Visible>
+    </EmbeddedResource>
+    <EmbeddedResource Include="$(MSBuildProjectDirectory)\Resources\Strings.fr.resx">
+      <Visible>True</Visible>
+    </EmbeddedResource>
+    <EmbeddedResource Include="$(MSBuildProjectDirectory)\Resources\Strings.it.resx">
+      <Visible>True</Visible>
+    </EmbeddedResource>
+    <EmbeddedResource Include="$(MSBuildProjectDirectory)\Resources\Strings.ja.resx">
+      <Visible>True</Visible>
+    </EmbeddedResource>
+    <EmbeddedResource Include="$(MSBuildProjectDirectory)\Resources\Strings.ko.resx">
+      <Visible>True</Visible>
+    </EmbeddedResource>
+    <EmbeddedResource Include="$(MSBuildProjectDirectory)\Resources\Strings.ru.resx">
+      <Visible>True</Visible>
+    </EmbeddedResource>
+    <EmbeddedResource Include="$(MSBuildProjectDirectory)\Resources\Strings.zh-Hans.resx">
+      <Visible>True</Visible>
+    </EmbeddedResource>
+    <EmbeddedResource Include="$(MSBuildProjectDirectory)\Resources\Strings.zh-Hant.resx">
+      <Visible>True</Visible>
+    </EmbeddedResource>
+  </ItemGroup>
```

Comment:
Given that we are importing these in the common targets do they still correctly show up in VS? What about for projects that don't have any or only partial of these files? Should we add some Exists conditions?